section1/exercises/dataTypes.js

```diff
@@ -58,16 +58,16 @@ console.log(1 === 2);
 console.log(7 > 2);
 
 // YOU DO: log to the console the result of "hello" is equal to "Hello":
-
+console.log("hello" === "Hello");
 // YOU DO: log to the console the result of 3 is not equal to 4:
-
+console.log(3 != 4);
 // YOU DO: log to the console the result of 4 is less than or equal to 5:
-
+console.log(4 <= 5);
 
 
 //-------------------
 // FINAL CHECK
 //-------------------
 
-// Did you run this file in your terminal to make sure everything printed out to the console 
-// as you would expect? 
\ No newline at end of file
+// Did you run this file in your terminal to make sure everything printed out to the console
+// as you would expect? I did not get errors; the one log that I am unsure about fully was the modulo.
```

Comment:
Ok I see - you'd actually want to reverse it so it's `54 % 10` - in that case, you'll log `4` since 4 is the remainder from 54 divided by 10.